### PR TITLE
Add workflow-related release notes

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 1]
```diff
@@ -56,6 +58,10 @@ weight: 88
 * We added a yellow warning bar for read-only documents in the modal editors.
 * We now show a yellow warning bar when editing read-only documents in pop-up editors like the [enumeration editor](/refguide/enumerations/).
 * We improved the layout of the domain model of the **System** module.
+* With the introduction of the state change events for workflows, the user tasks are now deleted from the database after completion, failure, or abort. This keeps the user task table size smaller and is better for performance.
+*  We now open workflow user task pages in the defined layout (allowing also pop-up pages).
```
Comment (MariaShaposhnikova) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md:62]: Is it related to Workflow Commons only?

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 2]
```diff
@@ -105,6 +111,10 @@ weight: 88
 * We fixed an issue that prevented some grids from having initially sorted columns.
 * We fixed an issue where a text box ignored a value filled by a password manager.
 * We fixed an issue where MxAssist Logic Bot generated an incorrect aggregate action for the calculate maximum and calculate minimum suggestions.
+* We fixed an issue in workflow user targeting using XPath which left the user task in a created state. When that happens now, the user task gets into the failed state (and so does the workflow).
```
Comment (MariaShaposhnikova) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md]: Do we need to add a more information on what was the undesired behaviour? I mean this part:
"We fixed an issue in workflow user targeting using XPath which left the user task in a created state."

File: content/en/docs/releasenotes/studio-pro/9/9.12.md (hunk 1 above)
Comment (arjenw) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md:62]: It is not related to Workflow Commons. It's standard Mendix page functionality (you can define the layout type in pages)

File: content/en/docs/releasenotes/studio-pro/9/9.12.md (hunk 2 above)
Comment (arjenw) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: You mean what the issue was that left the user task in a created state? That had to do with an incorrect XPath or an exception during executing the XPath.

Such failures resulted in a user task in a created state and the workflow in in-progress state. However, the workflow could not proceed out of this state (unless the issue was resolved). Now it properly shows up as failed with a helpful explanation.

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 3]
```diff
@@ -56,6 +58,10 @@ weight: 88
 * We added a yellow warning bar for read-only documents in the modal editors.
 * We now show a yellow warning bar when editing read-only documents in pop-up editors like the [enumeration editor](/refguide/enumerations/).
 * We improved the layout of the domain model of the **System** module.
+* With the introduction of the state change events for workflows, the user tasks are now deleted from the database after completion, failure, or abort. This keeps the user task table size smaller and is better for performance.
```
Comment (PaulJongenMx) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md]: Add this line before the rest. "In Mendix 9.10 the scope of the workflow domain model has changed to mainly support actively running tasks.".
Add line at the end: "State change events can be used to store historical WorkflowUserTask data."

File: content/en/docs/releasenotes/studio-pro/9/9.12.md (hunk 2 above)
Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: I will then add this explanation and write: We fixed an issue where incorrect XPath or an exception in user targeting left the user task in a created state. When that happens now, the user task gets into the failed state (and so does the workflow).
Does it look okay @arjenw ?

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 4]
```diff
@@ -56,6 +58,10 @@ weight: 88
 * We added a yellow warning bar for read-only documents in the modal editors.
 * We now show a yellow warning bar when editing read-only documents in pop-up editors like the [enumeration editor](/refguide/enumerations/).
 * We improved the layout of the domain model of the **System** module.
+* In Mendix 9.10, the scope of the workflow domain model has changed to mainly support actively running tasks. We now introduced the state change events for workflows, and the user tasks are now deleted from the database after completion, failure, or abort. This keeps the user task table size smaller and is better for performance. State change events can be used to store historical WorkflowUserTask data.
```
Comment (MariaShaposhnikova) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md:61]: @PaulJongenMx , I've rewritten the release note a bit and implemented your comment.

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 5]
```diff
@@ -21,6 +21,8 @@ weight: 88
 *  Did you ever want to be able to temporarily disable some activities in a microflow or nanoflow, because you are still building and testing your flow, for example? Now you can! When you right-click a microflow activity and select **Disable activity**, the activity is shown as faded out. When running your flow and while debugging, these disabled activities are skipped. You can also have disabled blocks of activities detached from your flow, without this resulting in consistency check errors. Try it out!
 	{{% alert type="info" %}}This is based on upvoted ideas from [Mathilde Mulder](https://forum.mendixcloud.com/link/ideas/257), [Erik Kuiper](https://forum.mendixcloud.com/link/ideas/1160), and [Ramón Frigge](https://forum.mendixcloud.com/link/ideas/993) submitted to the Mendix Idea Forum. Thanks, Mathilde, Erik, and Ramón!
 	{{% /alert %}}
+* We added the possibility to retry failed workflows. This would restart only the failed activities and put the Workflow in progress again. For more information, see [Change Workflow State](/refguide/change-workflow-state/).
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md]: Should "Workflow" be capitalized?

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 6]
```diff
@@ -21,6 +21,8 @@ weight: 88
...
+* We introduced a capability to track workflow and user task state changes using state change event microflows. They can be app-wide (in the [App Settings](/refguide/project-settings/)), and/or specific of a certain workflow (overriding the ones defined in **App Settings**). This feature can be used to build an audit system for     workflows and user tasks, for example.
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md]: Are there unnecessary extra spaces between "for" and "workflows"?

**Pull request merged**